core/src/main/java/com/yahoo/ycsb/BasicDB.java

```diff
@@ -80,7 +80,7 @@ public void init()
 		verbose=Boolean.parseBoolean(getProperties().getProperty(VERBOSE, VERBOSE_DEFAULT));
 		todelay=Integer.parseInt(getProperties().getProperty(SIMULATE_DELAY, SIMULATE_DELAY_DEFAULT));
 		randomizedelay=Boolean.parseBoolean(getProperties().getProperty(RANDOMIZE_DELAY, RANDOMIZE_DELAY_DEFAULT));
-		if (verbose)
+		if (verbose) synchronized(System.out)
```

Comment:
Please include the braces for the added nesting.

Does this help if another thread is printing something other than. Properties?